bytecomp/simplif.ml

```diff
@@ -686,9 +685,9 @@ let split_default_wrapper ?(create_wrapper_body = fun lam -> lam)
         (wrapper_body, (inner_id, inner_fun))
   in
   try
-    let wrapper_body, inner = aux [] body in
-    [(fun_id, Lfunction{kind; params; body = create_wrapper_body wrapper_body;
-       attr; loc}); inner]
+    let body, inner = aux [] body in
+    let attr = default_stub_attribute in
```

Comment:
It's not really discarding them, it's just not duplicating them onto the wrapper. Attributes which should appear on both the wrapper and the function seem less likely to me (there are none at the moment), so I think starting from `default_stub_attribute` and explicitly copying any that it should share would be more natural (which is what the current code does except that at the moment there are none to explicitly share).